### PR TITLE
adding binaries for Redmi note 7

### PR DESCRIPTION
**Pull request by index986**
### Reason

add supprot for redmi note 7
### Checklist

* [ x] Is what you changed Tested?
* [ x] Are the Patched Files Compiled?
* [x ] Did you Clean Up the Files?

### CODE REVIEW

**Review by Robotix22 — CHANGES_REQUESTED** [review]
Please run `cleanup.sh` once and Update the Pull Request

**Comment by index986** [comment]
idk if it got updated but i think i cleaned it up

File: lavender/DXE.inc
Comment (Robotix22) [inline lavender/DXE.inc:1]: Remove this File

File: lavender/DXE.dsc.inc
Comment (Robotix22) [inline lavender/DXE.dsc.inc:1]: Remove this File

File: lavender/CleanUp.sh
Comment (Robotix22) [inline lavender/CleanUp.sh:1]: Remove this File

File: lavender/APRIORI.inc
Comment (Robotix22) [inline lavender/APRIORI.inc:1]: Remove this File

**Pull request closed without merging**